### ISSUE TITLE
Invalid locale file "lang\\zh-CN.json"

### ISSUE THREAD

**Issue by li-xunhuan** [golang-module/carbon#111]
Hello,

I encountered an issue with the following code:
```go
// 倒计时
func TestDjs(t *testing.T) {
	lange := carbon.NewLanguage()
	lange.SetLocale("zh-CN")
	car := carbon.SetLanguage(lange)
	now := car.Now()

	offDutyTime := now.StartOfDay().AddHours(20)
	t.Logf("下班时间: %v", offDutyTime)
	t.Logf("距离下班还有: %v", offDutyTime.DiffInString(now))
	t.Log(now.Error)
	t.Log(offDutyTime.Error)
}
```

golang version: **such as 1.16**
1.17

carbon version: **such as 1.5.0**
v2.0.1

time zone: **such as Japan**
China

logs:
```
=== RUN   TestDjs
    all_test.go:63: 下班时间: 
    all_test.go:64: 距离下班还有: 
    all_test.go:65: invalid locale file "lang\\zh-CN.json", please make sure the json file exists and is valid
    all_test.go:66: invalid locale file "lang\\zh-CN.json", please make sure the json file exists and is valid
--- PASS: TestDjs (34.18s)
PASS
```

pic
![微信图片_20211218184301](https://user-images.githubusercontent.com/10476982/146638259-7be6909b-4ab1-48ca-9752-52ca188da333.png)

Thanks!

**Comment by gouguoyin** [comment]
我看下

**Comment by gouguoyin** [comment]
本地测试没有问题
![image](https://user-images.githubusercontent.com/13517412/149888092-37040dd0-cf5d-4a21-a5ae-4b19e034a6c6.png)

**Comment by gouguoyin** [comment]
window环境么？

**Comment by gouguoyin** [comment]
It has been fixed in v1.7.1 or v2.2.1
